library/include/hipsparse.h

```diff
@@ -8962,29 +8962,29 @@ typedef enum
     HIPSPARSE_COOMM_ALG2            = 2,
     HIPSPARSE_COOMM_ALG3            = 3,
     HIPSPARSE_CSRMM_ALG1            = 4,
-    HIPSPARSE_SPMM_ALG_DEFAULT      = 5,
-    HIPSPARSE_SPMM_COO_ALG1         = 6,
-    HIPSPARSE_SPMM_COO_ALG2         = 7,
-    HIPSPARSE_SPMM_COO_ALG3         = 8,
-    HIPSPARSE_SPMM_COO_ALG4         = 9,
-    HIPSPARSE_SPMM_CSR_ALG1         = 10,
-    HIPSPARSE_SPMM_CSR_ALG2         = 11,
-    HIPSPARSE_SPMM_BLOCKED_ELL_ALG1 = 12,
-    HIPSPARSE_SPMM_CSR_ALG3         = 13
+    HIPSPARSE_SPMM_ALG_DEFAULT      = 0,
+    HIPSPARSE_SPMM_COO_ALG1         = 1,
+    HIPSPARSE_SPMM_COO_ALG2         = 2,
+    HIPSPARSE_SPMM_COO_ALG3         = 3,
+    HIPSPARSE_SPMM_COO_ALG4         = 5,
+    HIPSPARSE_SPMM_CSR_ALG1         = 4,
+    HIPSPARSE_SPMM_CSR_ALG2         = 6,
+    HIPSPARSE_SPMM_CSR_ALG3         = 12,
+    HIPSPARSE_SPMM_BLOCKED_ELL_ALG1 = 13
```

Comment:
same here
I think cusparse removed parts of it in their latest release and renumbered.